src/SpecterWallet.ts

```diff
@@ -1,19 +1,21 @@
 import { createChainAddress, sendToChainAddress } from "lightning"
 import _ from "lodash"
-import { yamlConfig } from "./config"
 import { bitcoindAccountingPath, lndAccountingPath, lndFeePath } from "./ledger/ledger"
 import { getActiveOnchainLnd, lndsBalances } from "./lndUtils"
 import { MainBook } from "./mongodb"
 import { getOnChainTransactions } from "./OnChain"
+import { Logger, SpecterWalletConfig } from "./types"
 import { UserWallet } from "./userWallet"
 import { BitcoindClient, bitcoindDefaultClient, btc2sat, sat2btc } from "./utils"
 
 export class SpecterWallet {
   bitcoindClient
-  logger
+  readonly logger: Logger
+  readonly config: SpecterWalletConfig
 
-  constructor({ logger }) {
+  constructor({ logger, config }) {
```

Comment:
I wasn't sure if there was any way to include an 'ConstructorArgs' type here like you all did [at this line](https://github.com/GaloyMoney/galoy/pull/321/files#diff-0c83656ad06f75fb5b1c410eda5125a6fa79a5669ae0a211e174f76209c79813R14) before

Reply:
Based on call with `bodymindarts` I decided to skip this for now since the args here are finite and already typed.

Reply:
We can just do this -> `constructor({ logger, config }: { logger: Logger; config: SpecterWalletConfig })`
Or am I missing something?

Reply:
Ah yes, this is what I was missing thanks! I'm still finding my way with Typescript.

I can include it in another PR where I'm thinking to tighten up the typing for some constructor functions as well.